fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/mappers/PaginationMapper.java

```diff
@@ -46,8 +46,6 @@ public PaginationMapper(
      * @param resultSet  The result set to be cut down.
      *
      * @return The page of results desired.
-     *
-     * @throws com.yahoo.bard.webservice.web.PageNotFoundException if the page requested is past the last page of data.
```

Comment:
Updated checkstyle detected that this method is actually not throwing exception, thus removing it:

```
JavadocMethod: Unused @throws tag for 'com.yahoo.bard.webservice.web.PageNotFoundException'.
```